### PR TITLE
Standardize error contents, and remove extra brackets in arrow functions

### PR DESCRIPTION
**Pull request by kirinrastogi**
All error responses should now have the following properties:
`err: 'Description of the err as a String or an Object'`
`givens: {'An object that holds the data that was given'}`

~~I also removed the extra brackets in arrow functions when there is only 1 parameter in the article route only. I didn't want to create conflicts for Vikram, as he is modifying the Keywords route.~~
Done for `/keywords/:word`

### CODE REVIEW

**Comment by JackyChiu** [comment]
>I didn't want to create conflicts for Vikram

Survival of the fittest, JUST DO IT

File: routes/article.js [hunk 1]
```diff
@@ -13,24 +13,24 @@ router.post('/vote', (req, res) => {
...
-            reqBody: req.body
+            givens: req.body
         })
         return
     }
 
     let articleChanged = null
     Article.findById(req.body.id)
-        .then((article) => {
+        .then(article => {
```
Comment (JackyChiu) [inline routes/article.js]: You have the rule for braces messed up, unless it's a oneline express arrow function, you don't omit the braces. Checkout airbnb's style guide, we're trying to follow it (although we did throw out the semi-colons) 
https://github.com/airbnb/javascript#arrow-functions

File: routes/article.js [hunk 2]
```diff
@@ -13,24 +13,24 @@ router.post('/vote', (req, res) => {
     if (amount === 0) {
         res.status(500).send({
             err: "Invalid value for vote",
-            reqBody: req.body
+            givens: req.body
```
Comment (JackyChiu) [inline routes/article.js]: Let's also be open to other words that could describe the property just as well (I can't think of a better one rn though)

File: routes/keyword.js [hunk 3]
```diff
@@ -11,15 +11,18 @@ router.get('/:word', (req, res) => {
...
-        console.log(wordList[0])
-        console.log(wordList[0].votes)
-        console.log(word)
-        res.send(wordList[0].votes)
-    }).catch((err) => {
-        console.log(err)
-        res.status(500).end()
-    })
+    Keyword
+        .find({word})
```
Comment (JackyChiu) [inline routes/keyword.js]: `.find({ word })` pls 😄

File: routes/keyword.js [hunk 4]
```diff
@@ -11,15 +11,18 @@ router.get('/:word', (req, res) => {
...
+        .lean()
+        .then(wordList => {
+            console.log(wordList[0])
+            console.log(wordList[0].votes)
+            console.log(word)
```
Comment (JackyChiu) [inline routes/keyword.js]: Do you mind getting rid of all the logs for production as well :3

File: routes/article.js [hunk 5]
```diff
@@ -99,10 +93,10 @@ router.get('/new', (req, res) => {
 
     getNextArticles(findParams, sortParams)
         .then(articles => res.send(articles))
-        .catch((err) => {
+        .catch(err => {
```
Comment (JackyChiu) [inline routes/article.js]: Missed the braces here 😬

Comment (JackyChiu) [inline routes/article.js]: Or you can make it a one line express like this:
```javascript
.catch(err => (
  res.status(500).send({
    err: `Newest articles weren't found: ${err}`,
    givens: req.query
  }))
```

Style from https://github.com/airbnb/javascript#arrows--paren-wrap

File: routes/keyword.js (hunk 3 above)
Comment (JackyChiu) [inline routes/keyword.js]: This is still here 😅

**Comment by kirinrastogi** [comment]
I reverted and readded my changes

**Comment by JackyChiu** [comment]
🚢 💯

**Comment by kirinrastogi** [comment]
Closed #35

**Pull request merged**